projects/packages/image-cdn/tests/php/bootstrap.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * Bootstrap.
+ *
+ * @package automattic/
+ */
+
+/**
+ * Include the composer autoloader.
+ */
+require_once __DIR__ . '/../../vendor/autoload.php';
```

Comment:
Can we port the tests over at the same time? Photon is an area of Jetpack that does have a good amount of unit tests written 

ex: https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/tests/php/test_class.jetpack_photon.php

https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/tests/php/test_functions.photon.php

With moving/refactoring, we can lean on these to help us ensure we're keeping the functionality (or at least are aware of gaps) and prevent regressions.